### PR TITLE
Update tests readmes

### PR DESCRIPTION
**Pull request by HarelM**
Update some info on how to run tests and see the results.

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ npm test
 Note that for Linux you'll need to prefix test commands with `xvfb-run -a` for example `xvfb-run -a npm run test`
 Note that some tests rely on the built project and will not pass until you have run `npm run build-dist`.
 
-To run individual tests:
+To run specific tests:
 
- - Unit tests: `npx jest path/to/file.test.ts` (e.g. `npx jest src/style/style_layer.test.ts`)
+ - Unit test file: `npx jest path/to/file.test.ts --reporters=default` (e.g. `npx jest src/style/style_layer.test.ts`)
+ - Unit test with name: `npx jest path/to/file.test.ts --reporters=default -t "test name"`
```
Comment (sbachinin) [inline test/README.md:22]: I would rephrase it to something like "Run only those test cases that match the test name"

Comment (HarelM) [reply test/README.md]: I've updated the text.

File: test/README.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ npm test
 Note that for Linux you'll need to prefix test commands with `xvfb-run -a` for example `xvfb-run -a npm run test`
 Note that some tests rely on the built project and will not pass until you have run `npm run build-dist`.
 
-To run individual tests:
+To run specific tests:
 
- - Unit tests: `npx jest path/to/file.test.ts` (e.g. `npx jest src/style/style_layer.test.ts`)
+ - Unit test file: `npx jest path/to/file.test.ts --reporters=default` (e.g. `npx jest src/style/style_layer.test.ts`)
```
[the remark this answers is not in the document]
Comment (HarelM) [reply test/README.md]: It doesn't really matters... can be before or after.

[the remark this answers is not in the document]
Comment (sbachinin) [reply test/README.md]: I meant that it'd be better to have `--reporters=default` in both strings. But yes, not a big deal

**Review by sbachinin — APPROVED** [review]
